packages/hooks/src/createRunHook.js

```diff
@@ -22,10 +22,17 @@ function createRunHook( hooks, returnFirstArg ) {
 	 */
 	return function runHooks( hookName, ...args ) {
 		let handlers;
-		if ( hooks[ hookName ] ) {
-			handlers = hooks[ hookName ].handlers;
+		if ( ! hooks[ hookName ] ) {
```

Comment:
There's an identical condition below in this function which should have been removed as its now redundant (dead code, as the condition will never be satisfied).